ddtrace/internal/constants.py

```diff
@@ -8,5 +8,8 @@
     PROPAGATION_STYLE_B3,
     PROPAGATION_STYLE_B3_SINGLE_HEADER,
 )
-
+_TRACESTATE_KEY = "tracestate"
+_TRACEPARENT_KEY = "traceparent"
+_W3C_TRACESTATE_ORIGIN_KEY = "o"
+_W3C_TRACESTATE_SAMPLING_PRIORITY_KEY = "s"
```

Comment:
Any reason these are being added with underscore prefix in an internal module? Also, we could add the `W3C_` prefix for clarity.

```suggestion
W3C_TRACESTATE_KEY = "tracestate"
W3C_TRACEPARENT_KEY = "traceparent"
W3C_TRACESTATE_ORIGIN_KEY = "o"
W3C_TRACESTATE_SAMPLING_PRIORITY_KEY = "s"
```